cfpb/cfgov-refresh
# Setting up initial routes for conversion of ES sites

Setting up initial routes for conversion some ES pages

## Additions

- Added some empty index files for the flagged `/es/` routes.

## Changes

- Modified `cfgov/cfgov/urls.py` to add flag states for some `/es/` pages.

## Testing

1. Enable the `ES_CONV_FLAG` flag in Wagtail admin.
2. Visit some of the `/es/` urls that have been flagged.
3. Verify that you see nothing, and you eyes are wide shut. 
4. Verify that the `ask-cfpb` spanish pages are operational.

## Screenshots


## Notes

-

## Todos

-

## Checklist
